index.src.html

```diff
@@ -290,6 +290,16 @@
     </ul>
   </section>
 
+  <section algorithm="'create a credential' internal method">
+    <dfn for="Credential" method export>\[[Create]](data)</dfn> is called with a
+    {{CredentialCreationData}}, and returns either a {{Credential}}, if one can be created, or
```

Comment:
I'll try to find time to document the model I'm thinking of. Until then, go ahead with this, and we can fix it up later if I turn out to be convincing.